operatorapi/operator_tenants.go

```diff
@@ -1706,6 +1740,299 @@ func getTenantUsageResponse(session *models.Principal, params operator_api.GetTe
 	return info, nil
 }
 
+// getTenantLogsResponse returns the logs of a tenant
+func getTenantLogsResponse(session *models.Principal, params operator_api.GetTenantLogsParams) (*models.TenantLogs, *models.Error) {
+	// 30 seconds timeout
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	opClientClientSet, err := cluster.OperatorClient(session.STSSessionToken)
+	if err != nil {
+		return nil, prepareError(err, errorUnableToGetTenantLogs)
+	}
+
+	opClient := &operatorClient{
+		client: opClientClientSet,
+	}
+
+	minTenant, err := getTenant(ctx, opClient, params.Namespace, params.Tenant)
+	if err != nil {
+		return nil, prepareError(err, errorUnableToGetTenantLogs)
+	}
+	if minTenant.Spec.Log == nil {
+		retval := &models.TenantLogs{
+			Disabled: true,
+		}
+		return retval, nil
+	}
+	annotations := []*models.Annotation{}
+	for k, v := range minTenant.Spec.Log.Annotations {
+		annotations = append(annotations, &models.Annotation{Key: k, Value: v})
+	}
+	labels := []*models.Label{}
+	for k, v := range minTenant.Spec.Log.Labels {
+		labels = append(labels, &models.Label{Key: k, Value: v})
+	}
+	nodeSelector := []*models.NodeSelector{}
+	for k, v := range minTenant.Spec.Log.NodeSelector {
+		nodeSelector = append(nodeSelector, &models.NodeSelector{Key: k, Value: v})
+	}
+
+	if minTenant.Spec.Log.Db == nil {
+		minTenant.Spec.Log.Db = &miniov2.LogDbConfig{}
+	}
+
+	dbAnnotations := []*models.Annotation{}
+	for k, v := range minTenant.Spec.Log.Db.Annotations {
+		dbAnnotations = append(dbAnnotations, &models.Annotation{Key: k, Value: v})
+	}
+	dbLabels := []*models.Label{}
+	for k, v := range minTenant.Spec.Log.Db.Labels {
+		dbLabels = append(dbLabels, &models.Label{Key: k, Value: v})
+	}
+	dbNodeSelector := []*models.NodeSelector{}
+	for k, v := range minTenant.Spec.Log.Db.NodeSelector {
+		dbNodeSelector = append(dbNodeSelector, &models.NodeSelector{Key: k, Value: v})
+	}
+
+	if minTenant.Spec.Log.Audit == nil || minTenant.Spec.Log.Audit.DiskCapacityGB == nil {
+		minTenant.Spec.Log.Audit = &miniov2.AuditConfig{DiskCapacityGB: swag.Int(0)}
+	}
+
```

Comment:
```suggestion
	if minTenant.Spec.Log.Image == "" {
		minTenant.Spec.Log.Image = miniov2.DefaultLogSearchAPIImage
	}

	if minTenant.Spec.Log.Db.Image == "" {
		minTenant.Spec.Log.Db.Image = miniov2.LogPgImage
	}
```

Make sure to display the default logSearchApi and Postgres image if the user didn't defined custom images

Reply:
With this change now the image version is displayed:

<img width="703" alt="Screen Shot 2021-11-30 at 14 40 21" src="https://user-images.githubusercontent.com/1795553/144139785-8f01aea1-c1ea-4824-8e8c-6a517e7936e3.png">
<img width="627" alt="Screen Shot 2021-11-30 at 14 40 28" src="https://user-images.githubusercontent.com/1795553/144139782-7da23acc-8c6a-4a62-a22f-7676cafd4eed.png">
<img width="646" alt="Screen Shot 2021-11-30 at 14 40 31" src="https://user-images.githubusercontent.com/1795553/144139784-7f000112-e77e-4cc5-ac51-1678016fc925.png">

Besides that the feature works fine.